samples/Chat/public/index.html

```diff
@@ -7,7 +7,7 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <meta name="description" content="Web site created using create-react-app" />
+    <meta name="description" content="Web site created" />
```

Comment:
suggestion: add a better description here
```suggestion
    <meta name="description" content="Chat application powered by Azure Communication Services" />
```